### ISSUE TITLE
The two buttons "Back" and "Next" are displayed by screen readers before the table

### ISSUE THREAD

**Issue by TheOneRing** [owncloud/client#11779]
Please validate whether the focus is still on the button instead of on the spaces.

This is in the "Add Space" dialog.

**Comment by erikjv** [comment]
The current behaviour is different: when opening the modal view, there is no indication which widget has the focus. It is actually the spaces list, because hitting arrow-down will select the first space. But before doing this,  there is no focus/selection indicator. Hitting tab will show the focus going to the buttons, which might lead to confusion on what the first focused item is.

Maybe we should always select the first item in the list when we show the dialog, indicating where the focus is.

**Comment by erikjv** [comment]
On MacOS with VoiceOver:

The selected item is never read out.

**Comment by erikjv** [comment]
Note: check if it is clearer when we add a description to the list, indicating that the user should select a space.

**Comment by erikjv** [comment]
The original issue has been fixed. A separate issue (#11870) is created to track the initial selection issue.
